lib/plugins/tag/asset_link.js

```diff
@@ -18,8 +19,21 @@ module.exports = ctx => {
     const asset = PostAsset.findOne({post: this._id, slug});
     if (!asset) return;
 
-    const title = args.length ? args.join(' ') : asset.slug;
+    let escape = args[0];
+    if (escape === 'true' || escape === 'false') {
+      args.shift();
+    } else {
+      escape = 'false';
+    }
 
-    return `<a href="${url.resolve(ctx.config.root, asset.path)}" title="${title}">${title}</a>`;
+    let title = args.length ? args.join(' ') : asset.slug;
+    let attrTitle;
+    if (escape === 'true') {
+      attrTitle = title = escapeHTML(title);
+    } else {
+      attrTitle = escapeHTML(title);
```

Comment:
In retrospect, I think it's fine to disable escaping the `title`, but `attrTitle` should _always_ be escaped.